ts-client/src/dlmm/index.ts

```diff
@@ -4284,6 +4286,168 @@ export class DLMM {
     };
   }
 
+  /**
+ * The `seedLiquidity` function create multiple grouped instructions. The grouped instructions will be either [initialize bin array + initialize position instructions] or [deposit instruction] combination.
+ * @param
+ *    - `owner`: The public key of the positions owner.
+ *    - `seedAmount`: Lamport amount to be seeded to the pool.
+ *    - `price`: Price in UI format
+ *    - `base`: Base key
+ *    - `roundingUp`: 
+ * @returns {Promise<Transaction>}
+ */
+  public async seedLiquiditySingleBin(
+    owner: PublicKey,
+    base: PublicKey,
+    seedAmount: BN,
+    price: number,
+    roundingUp: boolean
+  ): Promise<Transaction> {
+    const toLamportMultiplier = new Decimal(
+      10 ** (this.tokenY.decimal - this.tokenX.decimal)
+    );
+
+    const pricePerLamport = new Decimal(price).mul(toLamportMultiplier);
+    let binId: BN = DLMM.getBinIdFromPrice(pricePerLamport, this.lbPair.binStep, true);
+    if (roundingUp) {
+      binId = DLMM.getBinIdFromPrice(pricePerLamport, this.lbPair.roundStep, false);
```

Comment:
what is `this.lbPair.roundStep`?

Reply:
it's my mistake, should be `this.lbPair.binStep`